src/engine/input/spike_matrix.c

```diff
@@ -0,0 +1,67 @@
+internal InputSpikeMatrix*
+input_spike_matrix_create(Memory* memory, u32 time_max, u32 n_neurons) {
```

Comment:
This naming is really bad, this is a SpikeTrain and it should be used as such,
we should be able to read it from a file and write it to a file

Reply:
only need reading for now